lib/widgets/buttons.dart

```diff
@@ -122,51 +122,192 @@ class _DropDownState extends State<DropDown> {
   Widget build(BuildContext context) {
     return Container(
       width: 185,
-      height: 56,
+      height: 51,
```

Comment:
from what i remember, we're using 56 as base height for all input fields and buttons but we can change this later, no need to worry about it